README.md

```diff
@@ -1,13 +1,56 @@
 # Springroll Seed
 
-Springroll Seed is intendend to be used as starting point for games projects. It comes bundled with Webpack, Babel, as well as a few other node modules to streamline development.
+SpringRoll Seed is intended to be used as a starting point for games projects. It comes bundled with Webpack, Babel, as well as a few other node modules to streamline development.
+
+## Setup
+
+```
+$ git clone https://github.com/SpringRoll/Springroll-Seed.git my_project
+
+$ cd my_project
+
+$ npm install
+```
 
 ## Commands
 
-## npm start
+SpringRoll Seed comes with three commands
+
+### npm start
 
 Starts the dev server
 
-## npm run build
+### npm run build:release
+
+Builds app for release
+
+### npm run build:debug
+
+Builds the app without mangling or minifying it for easier debugging
+
+# Dev Flow
+
+## Webpack
+
+SpringRoll Seed uses Webpack as its build process. If you are not familiar with Webpack, how it works is that it looks for a entry point, in Seed's case there are three options. `index.js`, `vendor.js`, and `styles.css`.
+
+All three entry points work the same way in that they will include and bundle any code or styles that are included inside them recursively and bundle them together into into one file for each entry point and will do it's best to minify the code along the way.
```

Comment:
"All three entry points work the same way. Each entry point includes and bundles any code or styles included inside itself recursively and bundle them together into one file. From there, it will attempt to minify the code along the way."